enhancements/console/dynamic-plugins.md

```diff
@@ -433,6 +433,35 @@ const VMHeading = () => {
 };
 ```
 
+In the `v1` API version, an additional field is introduced for enabling and loading i18n namespace that
+given dynamic plugin contains. The i18n namespace name follows `plugin__{plugin-name}` naming convention.
+If i18n is not enabled in the `ConsolePlugin` resource, no i18n namespaces will be loaded for the plugin.
+
+```yaml
+apiVersion: console.openshift.io/v1
+kind: ConsolePlugin
+metadata:
+  name: acm
+spec:
+  displayName: 'Advanced Cluster Management'
+  i18n:
+    enabled: true
+```
+
+In the 4.11 release, a `console.openshift.io/default-i18next-namespace` annotation
+is being introduced. The annotation indicates whether the `ConsolePlugin` contains
```

Comment:
Where is this annotation being introduced? Should plugins add it to the CSV? Maybe we should mention that.

Reply:
It is added to the `ConsolePlugin` resource.

Reply:
So the annotation will be checked for DynamicPlugin CR.